src/main/java/core/basesyntax/service/OperationStrategyImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.operation.OperationHandler;
+import java.util.Map;
+import java.util.NoSuchElementException;
+
+public class OperationStrategyImpl implements OperationStrategy {
```

Comment:
you can remove interface for Strategy class 
```suggestion
public class OperationStrategy {
```